src/OfflineWorkflowsSample/OfflineWorkflowsSample/Views/Browsing/PortalGroupView.xaml

```diff
@@ -0,0 +1,57 @@
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:offlineWorkflowSample="using:OfflineWorkflowSample"
+    xmlns:converter="using:OfflineWorkflowSample.Infrastructure.Converter"
+    mc:Ignorable="d"
+    NavigationCacheMode="Required">
+    <Page.Resources>
+        <converter:NullEnumToBlankStringConverter x:Key="NullEnumToBlankStringConverter" />
+    </Page.Resources>
+    <SplitView IsPaneOpen="True"
+               DisplayMode="Inline"
+               PaneBackground="Transparent">
+        <SplitView.Pane>
+            <ScrollViewer>
+                <StackPanel Padding="5,0,20,0">
+                    <ListView ItemsSource="{x:Bind PortalViewModel.VisibleGroups,Mode=OneWay}"
+                              SelectedItem="{x:Bind PortalViewModel.SelectedGroup,Mode=TwoWay}"
+                              Background="Transparent">
+                        <ListView.ItemTemplate>
+                            <DataTemplate x:DataType="offlineWorkflowSample:PortalFolderViewModel">
+                                <TextBlock Text="{x:Bind Title}" />
+                            </DataTemplate>
+                        </ListView.ItemTemplate>
+                    </ListView>
+                    <TextBlock Text="Filters" FontWeight="SemiBold" />
+                    <CheckBox IsChecked="{x:Bind PortalViewModel.OfflineOnlyFilter,Mode=TwoWay}"
+                              Margin="0,0,5,0"
+                              Content="Restrict to content available offline" />
+                    <SearchBox QueryText="{x:Bind PortalViewModel.SearchFilter,Mode=TwoWay}"
+                               PlaceholderText="Filter"
+                               Margin="0,5,0,5"
+                               SearchHistoryEnabled="False" />
+                    <ComboBox ItemsSource="{x:Bind PortalViewModel.AvailableTypeFilters,Mode=OneWay}"
+                              HorizontalAlignment="Stretch"
+                              SelectedItem="{Binding PortalViewModel.TypeFilter,Mode=TwoWay}">
+                        <!-- Not using x:Bind here because UWP binding can't handle a nullable x:DataType. -->
+                        <ComboBox.ItemTemplate>
+                            <DataTemplate>
+                                <TextBlock Text="{Binding Converter={StaticResource NullEnumToBlankStringConverter}}" />
+                            </DataTemplate>
+                        </ComboBox.ItemTemplate>
+                    </ComboBox>
+                </StackPanel>
+            </ScrollViewer>
+        </SplitView.Pane>
+        <SplitView.Content>
+            <GridView ItemsSource="{x:Bind PortalViewModel.SelectedGroup.Items,Mode=OneWay}"
+                      ItemTemplate="{StaticResource PortalItemTemplate}"
+                      SelectedItem="{Binding ViewModel.SelectedItem,Mode=TwoWay}"
```

Comment:
A little search quirk: Search for something, then open a webmap from search results. Now click on "Search" tab again and click on the same item again. App will stay on the search results page. Clicking the "back" button works fine in this scenario though.